runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -502,6 +502,10 @@ private LootTrackerBox buildBox(LootTrackerRecord record)
 			{
 				if (box.matches(record))
 				{
+					//float the matched box to the top of the UI list
+					logsContainer.remove(box);
```

Comment:
Since the common case will not require reordering you can optimize this via
```java
int idx = logsContainer.getComponentZOrder(box);
if (idx > 0) {
  logsContainer.remove(idx);
  logsContainer.add(box, 0);
}
```

Reply:
Good point! Implemented the change as recommended and verified it's still functioning as expected.